scripts/library.sh

```diff
@@ -498,6 +498,28 @@ function start_latest_knative_eventing() {
   start_knative_eventing "${KNATIVE_EVENTING_RELEASE}"
 }
 
+# Install Knative Eventing extension in the current cluster.
+# Parameters: $1 - Knative Eventing extension manifest.
+function start_knative_eventing_extension() {
+  header "Starting Knative Eventing Extension"
+  echo "Installing Extension CRDs from $1"
+  kubectl apply -f "$1"
+  # do the wait for running pods in specific extension function as the namespace could differ
```

Comment:
This is not the pattern used by the `start_knative_eventing` function. Are there scenarios that require eventing to be installed in a different namespace? If yes we may need to change them all.

Reply:
This is for the eventing extensions, the resources of some extensions are installed in other namespace, for example kafka-source (https://storage.googleapis.com/knative-nightly/eventing-contrib/latest/kafka-source.yaml), installs in `knative-sources` namespace. So if we'd add kafka setup util here later, it could check for resources in respective namespace. We could revert this and make `start_knative_eventing_extension` to look for running pods in $2. WDYT?

Reply:
Yeah I think that is better, you can pass `knative-eventing` in the function call.

Reply:
fixed. PTAL